cvmfs/sync_item.h

```diff
@@ -153,8 +162,11 @@ class SyncItem {
   mutable EntryStat scratch_stat_;
 
   bool whiteout_;
+  bool valid_graft_;
+  bool graft_marker_present_;
   std::string relative_parent_path_;
   std::string filename_;
+  ssize_t size_;
```

Comment:
perhaps better `grafted_size_` to clarify this is only used in the context of grafted files.